src/style.css

```
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:last-child {
+  margin: 0;
+}
+
+.stars-2 .stars_star:nth-child(1),
```

We can write it in shorter
```suggestion
.stars--1 :nth-child(1),
```

---

fixed